### PR TITLE
Update italian language for installer

### PR DESCRIPTION
**Pull request by bovirus**
@az0 

Please check and merge. Thanks.

### CODE REVIEW

**Comment by bovirus** [comment]
@az0 

Could you pelase merge this PR to enable me test the enw installer build for fix about italian language strings?
Note: I already added in 'NsisMultiUserLang.nsh' file the new strings for embeeded text that I will change in bleachbit.nsi.

The changes for "bleachbit.nsi" will be uplaoded after the etxt of txt strings..

Thanks.

**Comment by bovirus** [comment]
@az0 

Could you please check and merge? Thanks.

File: windows/NsisInclude/NsisMultiUserLang.nsh [hunk 1]
```diff
@@ -40,6 +40,25 @@ Version: 2019-11-11
 	LangString MULTIUSER_ELEVATION_NOT_SUPPORTED ${LANG_ENGLISH} "The operating system doesn't support elevation."
 	LangString MULTIUSER_LOGON_SERVICE_NOT_RUNNING ${LANG_ENGLISH} "Unable to elevate, Secondary Logon service not running."
 	LangString MULTIUSER_ELEVATION_ERROR ${LANG_ENGLISH} "Unable to elevate, error {ERROR}."
+	; text string for installer section
+	LangString SECTION_CORE ${LANG_ENGLISH} "Core (Required)"
+	LangString SECTION_SHORTCUTS ${LANG_ENGLISH} "Shortcuts"
+	LangString SECTION_START_MENU ${LANG_ENGLISH} "Start menu"
+	LangString SECTION_DESKTOP ${LANG_ENGLISH} "Dsktop"
```
Comment (az0) [inline windows/NsisInclude/NsisMultiUserLang.nsh:47]: One letter is missing

```suggestion
	LangString SECTION_DESKTOP ${LANG_ENGLISH} "Desktop"
```

**Review by az0 — APPROVED** [review]

File: windows/NsisInclude/NsisMultiUserLang.nsh (hunk 1 above)
Comment (az0) [reply windows/NsisInclude/NsisMultiUserLang.nsh:47]: @bovirus 

I merged it with this typo, and you can fix it in your next PR.

**Comment by az0** [comment]
The following warnings are new in the [AppVeyor build log](https://ci.appveyor.com/project/az0/bleachbit/builds/46075316?fullLog=true), though the space difference is less than 1KB.

```
6030: LangString "MUI_TEXT_FINISH_INFO_REBOOT" set multiple times for 1040, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1025)
6030: LangString "MUI_INNERTEXT_LICENSE_BOTTOM_RADIOBUTTONS" set multiple times for 1040, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1027)
6030: LangString "DESC_SECTION_CORE" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1039)
6030: LangString "DESC_SECTION_SHORTCUTS" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1040)
6030: LangString "DESC_SECTION_START_MENU" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1041)
6030: LangString "DESC_SECTION_DESKTOP" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1042)
6030: LangString "DESC_SECTION_QUICK_LAUNCH" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1043)
6030: LangString "DESC_SECTION_TRANSLATION" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1044)
6030: LangString "DESC_SECTION_SHRED" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1045)
6030: LangString "DESC_SECTION_SECTION_UNINSTALL" set multiple times for 1033, wasting space (.\NsisInclude\NsisMultiUserLang.nsh:1046)
```

**Comment by bovirus** [comment]
@az0 

The warning arecorrect.
But are not a problem.
I made some changes that overwrite current string base Italian language of NSIS.
Your version of NSIS seems not update (and in new NSIS version theer are some language fixes).
To avoid to force you to update NSIS I made the changes of some base italian strings and warnings advise that there two different point of teh string definitoon: one in base language and one in the script (and are newser then base italian language).

**Comment by az0** [comment]
>Your version of NSIS seems not update.

Yes, it is NSIS 3.03, while 3.08 was released September 2021. Feel free to open an issue for this, if you think it will help.

>Please compile the installer, post the link to download it, and I will check if all will be ok.

After every commit to Git, the new release is posted to the [CI build site](https://ci.bleachbit.org/). There is more detail on the [AppVeyor site](https://ci.appveyor.com/project/az0/bleachbit/), which contains the build log and info about failed builds.

**Comment by az0** [comment]
@bovirus 

Regarding a potential regression, would you please see this issue https://github.com/bleachbit/bleachbit/issues/1469 and respond there?

**Comment by bovirus** [comment]
@az0 

Is not necessaru no one regression.
The user that report the problem use a very old version that has the isuuee but is not this latest version that I requested to merge.,
I tested the latetst build (4.2.22416) and is working fine.
No error about translation or empty module.

**Pull request merged**